### ISSUE TITLE
Error appears when drop an STS file (.3ds)

### ISSUE THREAD

**Issue by YuF12345** [yuksk/SIDAM#42]
I'm a user of igor 9 (Version: 9.0.1.2) 
I have two problems which may be related with each other.

1. Error message
I installed SIDAM into igor without problem.
When I drop an STS data (*.3ds), it shows an error saying 
"While executing a wave write, the following error occurred: Index out of range for wave "_free_".

Do you have any idea what is this?
It appears even I open the data from menubar->Load Data.
It didn't appear for simple topograph data (*.sxm)

2. According to your Gif movie of "Spectrum viewer", one can check point spectrum even if the cursor is on topograph.
I can see point spectrum when the cursor is on dI/dV map, but cannot see it on topograph.
Is there anything I need to do to connect these two images such as sync?

**Comment by yuksk** [comment]
It is hard to diagnose what's happening only from the given information. If you can tell me the line number causing the problem, it may help to diagnose the problem. To pin down the line number, you need to turn on the debugger of Igor Pro and select SIDAM > Developer > Show SIDAM Procedures.

The second one is a separate issue. Please raise another issue with an appropriate title so that people who have the same question can refer to it later.

**Comment by yuksk** [comment]
See #45 for the answer to the second question.

**Comment by BRMSm** [comment]
Hello, I'm a user of SIDAM (and also a colleague of YuF12345).

I've replicated the error in 1. with debugging turned on, and it seems that the "_free_" wave that is throwing an error is on line 361 of the ipf "SIDAM_LoadData_Nanonis_3ds.ipf" I stepped through the function with the debugger, and it seems that the problem is that it is trying to assign refw[1,]=avgw[p-1], but avgw has 0 points at the time of assignment (giving the out of bounds error). I guess the problem is actually on line 358 where the avgw wave reference wave is created/assigned. Unfortunately I wasn't able to figure out more than that due to being a relative novice in igor programming. 

Hopefully this helps debug the issue.
